app/packages/looker-3d/src/fo3d/MediaTypeFo3d.tsx

```diff
@@ -376,6 +376,132 @@ export const MediaTypeFo3dComponent = () => {
     [onChangeView]
   );
 
+  // zoom to selected labels and use them as the new lookAt
+  useHotkey(
```

Comment:
Could we move this to an independent hooks and file? I think this would help readability and future maintenance